#! /usr/bin/env python
#! -*- coding: utf-8 -*-
PACKAGE = "mapvel"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("pathnum",double_t, 0, "the path num",4,1,8)
gen.add("sureStopLine",double_t, 0, "sure draw the stopline",1,0,1)
gen.add("lan_width",double_t, 0, "the path width",3.2,2.0,5.0)
gen.add("lan_long",double_t, 0, "the path long",400,0,1000)
gen.add("lan_halflong",double_t, 0, "the half path long",200,0,500)
gen.add("org_x",double_t, 0, "the  x coodration of org",-6.0,-100,100)
gen.add("org_y",double_t, 0, "the  y coodration of org",0,-100,100)
gen.add("theta_deg",double_t, 0, "the clockwise rotation angel of path",0,-45,45)
gen.add("interval",double_t, 0, "the interval of equidistant line",25,0,100)

exit(gen.generate(PACKAGE,"mapvel","dynamicLoad"))
